pkg/cmd/recoverycontroller/cmd.go

```diff
@@ -116,6 +117,27 @@ func (o *Options) Run(ctx context.Context) error {
 		return err
 	}
 
+	resourceSyncController := resourcesynccontroller.NewResourceSyncController(
```

Comment:
find a way to restructure so that we can re-use the same controller for operator and here.